src/shared/types/variation-api/variantStudyPopulation.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ValueSetMetadata } from '../thoas/metadata';
+
+export type VariantStudyPopulation = {
+  name: string;
+  size: number;
+  description: string;
+  type: ValueSetMetadata;
+  global: boolean; // should this be named is_global instead?
```

Comment:
`is_global` sounds better. Will the BE change it though?

Reply:
We'll see. I left a comment about this in the [proposed variation data model](https://github.com/Ensembl/ensembl-vdm-docs/pull/1)

Reply:
variation team just pushed:

![image](https://user-images.githubusercontent.com/6834224/231474849-90727b44-fa37-474a-9488-9577bcb6a1ce.png)

🎉